inverse_covariance/tests/rank_correlation_test.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+from sklearn.utils.testing import assert_array_almost_equal
+
+from inverse_covariance.rank_correlation import (
+    _compute_ranks,
+    spearman_correlation,
+)
+
+Y = np.ones(shape=[10, 2])
+X = .001 * np.random.randn(20, 1)
+X = np.append(X, X, axis=1)
+
+
+def test_compute_ranks():
+    Y1 = 5.5 * Y
+    Y2 = _compute_ranks(Y)
+    assert_array_almost_equal(Y1, Y2)
+
+
+def test_spearman_correlation():
```

Comment:
I wouldn't worry too much about retesting any thing wrappers around `scipy.stats` functions but if there is something substantial enough and worth it, then that would be good when possible.